pkg/s3presign/sts_test.go

```
@@ -0,0 +1,271 @@
+				t.Errorf("unexpected RoleArn: %q", r.FormValue("RoleArn"))
+			}
+			if r.FormValue("RoleSessionName") != "test-session" {
+				t.Errorf("unexpected RoleSessionName: %q", r.FormValue("RoleSessionName"))
+			}
+			if r.FormValue("WebIdentityToken") != "my-token-content" {
+				t.Errorf("unexpected WebIdentityToken: %q", r.FormValue("WebIdentityToken"))
+			}
+
+			w.Header().Set("Content-Type", "text/xml")
+			w.WriteHeader(http.StatusOK)
+			w.Write([]byte(`<AssumeRoleWithWebIdentityResponse>
```

golangci-lint (2.11.4)

[error] Error return value of `w.Write` is not checked

(errcheck)